documentation/tutorial.md

```diff
@@ -109,6 +109,49 @@ some functions:
 3. parse_item(): function which will be called to handle response
    downloaded by Crawly. It must return the `Crawly.ParsedItem` structure.
 
+As it is, the init function returns an empty array for `start_urls` which we will
+fix now. If you already know what urls, you want to start crawling, you can place them in
+the array, else you can pick the urls you want to start with from a particular page
+on your website of interest, in this case `https://books.toscrape.com`.
+
+Go ahead and modify your `init` to look this way.
+
+```elixir
+  @impl Crawly.Spider
+  def init() do
```

Comment:
the init function should just provide the starting url and should be simpler 

(this is a quickstart afterall)

Keep the document parsing logic under `parse_item`